### ISSUE TITLE
[R-package] add flag of displaying train loss for lgb.cv()

### ISSUE THREAD

**Issue by mayer79** [microsoft/LightGBM#4911]
In https://github.com/microsoft/LightGBM/issues/1105, it was suggested to add the option `eval_train_metric` to `lgb.cv()` in order to see not only CV scores but also training scores. This is very useful to monitor overfitting. 

In Python, it was implemented in https://github.com/microsoft/LightGBM/pull/2089. 

I would love to see something similar for R.

**Comment by jameslamb** [comment]
Thanks very much for the suggestion @mayer79 ! I support it.

Are you interested in contributing this? I'd be happy to answer any questions you have about where to make changes.
